Makefile

```diff
@@ -189,3 +194,9 @@ specific_test_local:: install_nodejs_sdk test_build
 
 dev:: lint build_nodejs
 test:: test_nodejs
+
+test_provider:
+	@echo ""
+	@echo "== test_provider ==================================================================="
+	@echo ""
+	cd provider && go test -v -short ./... -parallel $(TESTPARALLELISM)
```

Comment:
should this be run from CI? (or maybe it's already run somewhere else?)

Reply:
The tests are already run via the regular, previous test suite. The upgrade test machinery lives in `/provider` but the test programs themselves are the old ones at `/examples`. 

@t0yv0 I believe you introduced this Make target. Did you have anything particular scenario in mind?

Reply:
Yeah in ci-mgmt bridged providers this runs in prerequisites - https://github.com/pulumi/pulumi-aws/actions/runs/6949018124/job/18906240483#step:11:34 - the intention is to run some quick fast running tests as early as possible, before building SDKs and splitting the test matrix by language for the more slow running tests.